Assets/Plugins/VRWorks/Scripts/VRWorks.cs

```diff
@@ -143,7 +143,7 @@ public void OnPreRender()
       if (m_Camera.stereoEnabled)
       {
         // target size needs to be twice for single pass stereo
-        targetSize = (uint)((UnityEngine.VR.VRSettings.eyeTextureWidth + 0 )<< 17) | (uint)m_Camera.pixelHeight;
+        targetSize = (uint)((UnityEngine.XR.XRSettings.eyeTextureWidth + 0 )<< 17) | (uint)m_Camera.pixelHeight;

```

Comment:
I'm not sure how VR's are changed to XR's, but they seemed to cause the merge conflicts